api/src/main/java/net/kyori/adventure/sound/SoundStop.java

```diff
@@ -175,6 +175,18 @@
     };
   }
 
+  /**
+   * Stops a specific sound.
+   *
+   * @param sound the sound
+   * @return a sound stopper
+   * @since 4.8.0
+   */
+  static @NotNull SoundStop sound(final @NotNull Sound sound) {
```

Comment:
I think we can remove this method - just call `namedOnSource` from within `SoundImpl`